vuepress/docs/next/tutorials/devops/caching-and-clustering.md

```diff
@@ -4,131 +4,132 @@ The tutorials below cover the basic steps to setup and validate a clustered inst
 
 ```
-entando.k8s.operator.default.clustered.storage.class: "[your clustered RWX storage class]"
-entando.k8s.operator.default.non.clustered.storage.class: "[your RWO storage class]"
+entando.k8s.operator.default.clustered.storage.class: [your clustered RWX storage class]
+entando.k8s.operator.default.non.clustered.storage.class: [your RWO storage class]
 ```
 
-Set the values of both to the appropriate storage class for your configuration
+Set the values of both to the appropriate storage class for your configuration.
 
 ::: tip
 You can also scale an Entando Application without clustered storage using a `ReadWriteOnce (RWO)` policy by ensuring that the
 instances are all scheduled to the same node. This can be accomplished using taints on other nodes. Be aware of the pros and cons of scheduling
-instances to the same node. This will give you protection if the application instance itself dies or becomes unreachable and will help
-you get the most utilization of node resources. However, if the node dies or is shutdown you will have to wait for Kubernetes to reschedule the pods to a different node and your application will be down.
+instances to the same node. This will give you protection if the application instance itself dies or becomes unreachable, and will help
+you get the most utilization of node resources. However, if the node dies or is shutdown, you will have to wait for Kubernetes to reschedule the pods to a different node and your application will be down.
 :::
 
 ## Clustering
 
-This tutorial reviews setting up a clustered Entando App Engine using the default Infinispan Library Mode deployment that ships with the quickstart App Engine in the `entando-de-app`. The goal of the tutorial is to deploy a clustered instance of the App Engine and verify that we have a high availability and scalable deployment of the application.
+This tutorial reviews setting up a clustered Entando App Engine in the `entando-de-app`. The goal is to deploy a clustered instance of the App Engine and verify that we have a high availability and scalable deployment of the application.
 
 ### Prerequisites
-- An existing deployment of an Entando App or the ability to create a new one
-    - If you haven't created a deployment yet or don't have a yaml file for an Entando deployment follow the quickstart [here](../../../docs/getting-started/).
-- The Entando deployment must use an RDBMS. Clustered instances will not work correctly with in memory databases and a `dbms: none` configuration.
+- An existing deployment of an Entando App or the ability to create a new one.
+    - If you haven't created a deployment yet or don't have a yaml file for an Entando deployment, follow the [Quickstart instructions here](../../../docs/getting-started/).
+- The Entando deployment must use a RDBMS, a relational database system in which data is organized in a table structure. Clustered instances will not work correctly with in-memory databases and a `dbms: none` configuration.
```

Comment:
Usually "an RDBMS" since you'd say "an R-D-B-M-S" and spoken R starts with a vowel?

Reply:
"create a new one" is kind of redundant... very minor

i think we decided on YAML as a house rule

per our style guide meeting, remove "here" in link

"tabular" instead of "table"

just to clarify: "...databases **and** a `dbms: none`..." or "...databases **or** a `dbms: none`..."